draft-ietf-tls-dtls-connection-id.md

```diff
@@ -87,8 +87,8 @@ these values are insufficient. This is a particular issue in the Internet of Thi
 when devices enter extended sleep periods to increase their battery lifetime. The
 NAT rebinding leads to connection failure, with the resulting cost of a new handshake.
 
-This document defines an extension to DTLS to add a connection ID (CID) to the
-DTLS record layer. The presence of the connection ID is negotiated via a DTLS
+This document defines an extension to DTLS 1.2 to add a Connection ID (CID) to the
```

Comment:
Why is "Connection ID (CID)" used instead of the above already introduced "CID" ?